.github/workflows/update-golang-version.yml

```diff
@@ -0,0 +1,43 @@
+name: "update-golang-versions"
+on:
+  workflow_dispatch:
+  schedule:
+    # every Monday at around 2 am pacific/9 am UTC
+    - cron: "0 9 * * 1"
+env:
+  GOPROXY: https://proxy.golang.org
+  GO_VERSION: 1.16.4
+jobs:
+  bump-k8s-versions:
+    runs-on: ubuntu-20.04
```

Comment:
Using `ubuntu-latest` will prevent us from having to bump Ubuntu version in the future.